docs/Troubleshooting.md

```diff
@@ -11,6 +23,52 @@ $ bundle exec spring stop
 
 Run shopify_app generator again.
 
-### App installation fails with 'The page you’re looking for could not be found' if the app was installed before
+## App installation
+
+## My app won't install
```

Comment:
Nit: can we use ### for the subsection here? It looks like `App installation` is an empty section as it is.

Reply:
Good catch!